lighthouse-core/report/report-generator.js

```diff
@@ -104,6 +103,27 @@ class ReportGenerator {
       }
       return number;
     });
+
+    // value is boolean?
+    Handlebars.registerHelper('is-bool', value => (typeof value === 'boolean'));
+
+    // a > b
+    Handlebars.registerHelper('gt', (a, b) => (a > b));
+
+    // !value
+    Handlebars.registerHelper('not', value => !value);
+
+    // arg1 && arg2 && ... && argn
+    Handlebars.registerHelper('and', (...args) => {
```

Comment:
travis is failing b/c node 5 doesn't support the spread operator in function args. You'll have to use `arguments`.